README.md

```diff
@@ -165,4 +179,27 @@ for ls in to_remove:
 
 vis.show_gif()
 ```
-![removal](https://github.com/aghbit/Algorytmy-Geometryczne/assets/115979017/95c5e2d9-391a-4a03-b2a3-9351f8d5e8ec)
+![removal](https://github.com/aghbit/Algorytmy-Geometryczne/assets/115979017/026d2e8e-a756-4448-a4b4-55a40b6063dc)
+
+## Struktura repozytorium
+Całe repozytorium składa się z sześciu folderów:
+<li>lab1</li>
+<li>lab2</li>
+<li>lab3</li>
+<li>lab4</li>
+<li>tests</li>
+<li>visualizer </li>
+
+#### lab1-4
+Tutaj znajdują się przygotowane notebooki które musisz wypełnic. Każdy folder odpowiada jednemu laboratorium
```

Comment:
Jak już piszemy jakieś dłuższe zdania, to może warto, żeby były poprawne interpunkcyjnie. Tutaj brakuje  przecinka przed "które". W kolejnych linijkach również brakuje kilku przecinków. Polecam [languagetool](https://languagetool.org/pl).